tests/functional/test_tracing.py

```diff
@@ -212,3 +214,15 @@ def handler(event, context):
     result = handler({}, {})
     assert "testresult" in result
     assert "testresult2" in result
+
+
+def test_aiohttp_trace_config():
+    aiohttp_mock = Mock()
+    sys.modules["aiohttp"] = aiohttp_mock
+    from aws_lambda_powertools.tracing import aiohttp_trace_config
+
+    aiohttp_trace_config()
+
+    from aws_xray_sdk.ext.aiohttp.client import aws_xray_trace_config
+
+    assert aws_xray_trace_config.__doc__ == "aiohttp extension for X-Ray (aws_xray_trace_config)"
```

Comment:
What's going on here? not sure I follow

Reply:
@heitorlessa  we are just not hitting the part where we modify the `__doc__` (i am not sure why this is happening either, i was hoping you would know)

Reply:
We're basically only injecting the doctstring for IDE Intellisense when people are about to import that, nothing else. Testing it can be called or simply ignored from test coverage altogether suffices tbh